### PR TITLE
Make shelley cli names consistent

### PR DESCRIPTION
**Pull request by Jimbo4350**

### CODE REVIEW

File: cardano-cli/src/Cardano/CLI/Shelley/Parsers.hs [hunk 1]
```diff
@@ -130,12 +144,20 @@ pAddressCmd =
...
   VerificationKeyFile <$>
-    Opt.strOption
-      (  Opt.long "payment-verification-key-file"
-      <> Opt.metavar "FILE"
-      <> Opt.help "Filepath of the payment verification key."
-      <> Opt.completer (Opt.bashCompleter "file")
-      )
+    ( Opt.strOption
+        (  Opt.long "payment-verification-key-file"
+        <> Opt.long "verification-key-file"
```
Comment (intricate) [inline cardano-cli/src/Cardano/CLI/Shelley/Parsers.hs:149]: Did you intend to specify two of these?

File: cardano-cli/src/Cardano/CLI/Shelley/Parsers.hs [hunk 2]
```diff
@@ -926,22 +1003,35 @@ pAddress =
 pStakingVerificationKeyFile :: Parser VerificationKeyFile
 pStakingVerificationKeyFile =
```
Comment (intricate) [inline cardano-cli/src/Cardano/CLI/Shelley/Parsers.hs:1004]: Rename to `pStakeVerificationKeyFile`:

```suggestion
pStakeVerificationKeyFile :: Parser VerificationKeyFile
pStakeVerificationKeyFile =
```

File: cardano-cli/src/Cardano/CLI/Shelley/Parsers.hs [hunk 3]
```diff
@@ -757,12 +796,35 @@ pPoolId =
...
+        <> Opt.completer (Opt.bashCompleter "file")
+        )
+    <|>
+      Opt.strOption
+        (  Opt.long "private-key"
+        <> Opt.internal
+        )
+
+    )
+-- TODO: STOPPED HERE: cardano-cli shelley node key-gen-KES
```
Comment (intricate) [inline cardano-cli/src/Cardano/CLI/Shelley/Parsers.hs:812]: Remove TODO:

```suggestion

```

File: cardano-cli/src/Cardano/CLI/Shelley/Parsers.hs [hunk 4]
```diff
@@ -861,12 +930,20 @@ pTxFee =
...
+         <> Opt.completer (Opt.bashCompleter "file")
+         )
+    <|>
+       Opt.strOption
+         (  Opt.long "tx-body-file"
+         <> Opt.internal
+         )
+
+    )
+
```
Comment (intricate) [inline cardano-cli/src/Cardano/CLI/Shelley/Parsers.hs:946]: This is problematic as the help text will also denote `out-file` for commands that use `pTxBodyFile` to parse an input file.

File: cardano-cli/src/Cardano/CLI/Shelley/Parsers.hs [hunk 5]
```diff
@@ -861,12 +930,20 @@ pTxFee =
...
 
 pTxFile :: FileDirection -> Parser TxFile
 pTxFile fdir =
```
Comment (intricate) [inline cardano-cli/src/Cardano/CLI/Shelley/Parsers.hs:949]: When being used as an output file, the option name should be `out-file`.

**Comment by dcoutts** [comment]
bors merge

**Pull request merged**